recipes/opencolorio/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+from conans import ConanFile, CMake, tools
+import os, shutil
+
+class OpenEXRConan(ConanFile):
+    name = "opencolorio"
+    description = "A color management framework for visual effects and animation."
+    license = "BSD-3-Clause"
+    homepage = "https://opencolorio.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake", "cmake_find_package"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
```

Comment:
I assume patching was required and including opencolorio using `add_subdirectory` didn't work?

```suggestion
    exports_sources = ["patches/*"]
```

Reply:
Yes, the build system uses `CMAKE_SOURCE_DIR` and `CMAKE_BINARY_DIR` :( I forgot to remove `CMakeLists.txt` from `exports_sources`, it's corrected now.

Reply:
@p-podsiadly Too late, but I think that replacing by patch `CMAKE_SOURCE_DIR` and `CMAKE_BINARY_DIR` in CMakeLists (+ report to upstream) and using CMakeLists wrapper would have been a better option for recipe's maintenance.